### PR TITLE
Access admin page

### PR DESCRIPTION
**Pull request by MlulekiN**

### CODE REVIEW

File: src/test/java/pages/CommonAdminPage.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import utils.ConfigFileReader;
+
+import java.time.Duration;
+
+public class CommonAdminPage {
```
Comment (benedictnkosi) [inline src/test/java/pages/CommonAdminPage.java]: change this to LoginPage

File: src/test/java/stepDefinition/GeneralAminSteps.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package stepDefinition;
+
+import cucumber.TestContext;
+import io.cucumber.java.en.Given;
+import managers.PageObjectManager;
+import org.openqa.selenium.WebDriver;
+import pages.CommonAdminPage;
+import utils.ConfigFileReader;
+
+public class GeneralAminSteps {
```
Comment (benedictnkosi) [inline src/test/java/stepDefinition/GeneralAminSteps.java]: spelling error GeneralAminSteps

Comment (benedictnkosi) [reply src/test/java/stepDefinition/GeneralAminSteps.java]: change this to login steps

**Review by benedictnkosi — CHANGES_REQUESTED** [review]

**Review by benedictnkosi — APPROVED** [review]

File: src/test/java/pages/CommonAdminPage.java (hunk 1 above)
Comment (MlulekiN) [reply src/test/java/pages/CommonAdminPage.java]: Done

File: src/test/java/stepDefinition/GeneralAminSteps.java (hunk 2 above)
Comment (MlulekiN) [reply src/test/java/stepDefinition/GeneralAminSteps.java]: Done

**Pull request merged**